lib/utils.js

```diff
@@ -1,6 +1,8 @@
 const loadPartialConfig = require('@babel/core').loadPartialConfig
 const createTransformer = require('ts-jest').createTransformer
```

Comment:
This will still error if users don't have TypeScript installed, can you move the require to inside the `getTSConfig` function